R/app_alphaDiv.R

```diff
@@ -65,19 +65,19 @@ alphaDiv <- function(df, recordIdColumn, method = c('shannon','simpson','evennes
       
       attr <- list('computationDetails' = computeMessage,
                    'parameters' = character())
+
```

Comment:
The following bit sets a dummy variable metadata variable in case of an error with the alpha div calcs. Is it necessary? Can we instead return an empty list?